lecture_2/homework/app/models/warrior.rb

```diff
@@ -0,0 +1,14 @@
+class Warrior < ApplicationRecord
+  belongs_to :clan
+  belongs_to :defensible, polymorphic: true
```

Comment:
Jak chcesz spróbować swoich sił w nieco trudniejszym zadaniu to polecam przerobić budynki w taki sposób, żeby korzystały z STI (podobnie jak wojownicy). Pozwoli nam to np. użyć jednego wspólnego serializera do wszystkich budynków, ale będzie wymagało trochę walki z Railsami, bo połączenie polimorfizm + STI nieco przerasta domyślne zachowanie ActiveRecordu 😱